src/main.cpp

```diff
@@ -156,6 +160,10 @@ std::optional<std::string> handle_message(const MessageBuffer& message_buffer, A
 {
     json body = message_buffer.body();
 
+    if (body["method"] == "initialized") {
+        return std::nullopt;
```

Comment:
Is it proper to just return this? I haven't worked with the spec for some time but maybe we should also remember that we're initialized?

Reply:
Admittedly I've only looked at 2 lsp's so far (and started a perl one that barely works). All of them hook into initialize (as is done below this) and then just ignore initialized. Here's some ref https://microsoft.github.io/language-server-protocol/specification#initialized. 

Should I hook this into the if/else chain stylistically? I just realized i threw this in the top without considering its location relative to other handlers.

Reply:
Nah, seems fine then.